features/login/impl/src/main/kotlin/io/element/android/features/login/impl/screens/changeaccountprovider/ChangeAccountProviderView.kt

```diff
@@ -109,6 +109,7 @@ fun ChangeAccountProviderView(
                 // Other
                 AccountProviderView(
                     item = AccountProvider(
+                        url = "",
```

Comment:
Ideally this should be reworked a bit, by encapsulating `AccountProvider` in a sealed interface to properly handle the `Other` case.